lib/tsconfig/3.8.x/ng9/README.md

```diff
@@ -0,0 +1,20 @@
+<h1 align="center">
+   code style - TypeScript 3.8.x + Angular 9
+</h1>
+
+## About
+
+This sub-package hosts the [TypeScript](https://www.typescriptlang.org/) configuration for TypeScript 3.5.x. and Angular 9.
```

Comment:
`TypeScript 3.8.x`

Reply:
It seems that my search and replace didn't work well ^^
Thanks for catch !